public/app/features/datasources/components/picker/DataSourcePicker.tsx

```diff
@@ -192,14 +193,6 @@ export function DataSourcePicker(props: DataSourcePickerProps) {
     const sub = keyboardEvents.subscribe({
       next: (keyEvent) => {
         switch (keyEvent?.code) {
-          case 'ArrowDown':
-            openDropdown();
```

Comment:
We don't need to open the dropdown for Up/Down arrow keys, the dropdown is already open.